cirq-superstaq/cirq_superstaq/service.py

```diff
@@ -973,6 +973,86 @@ def submit_aces(
             lifespan=lifespan,
         )
 
+    def submit_cb(
+        self,
+        target: str,
+        shots: int,
+        circuits: cirq.Circuit,
+        num_channels: int,
+        num_sequences: int,
+        depths: list[int],
+        method: str | None = None,
+        noise: str | cirq.NoiseModel | None = None,
+        error_prob: float | tuple[float, float, float] | None = None,
+    ) -> str:
+        """Submits the jobs to characterize `target` through the Cycle Benchmarking protocol.
+
+        The protocol in detail can be found in: https://arxiv.org/abs/1902.08543.
+
+        Args:
+            target: The device target to characterize.
+            shots: How many shots to use per circuit submitted.
+            circuits: The process circuit to use in the protocol.
+            num_channels: The number of random Pauli decay channels to approximate error.
+            num_sequences: Number of circuits to generate per depth.
+            depths: Lists of depths representing the depths of Cycle Benchmarking
+                circuits to generate.
```

Comment:
nits (ditto in superstaq_client)
```suggestion
            num_sequences: The number of circuits to generate per depth.
            depths: List of depths of Cycle Benchmarking circuits to generate.
```